src/CreateQuiz.js

```diff
@@ -120,14 +124,15 @@ class CreateQuiz extends React.Component {
                     </Form.Group>
                     {this.state.questions.map((val, index) => {
                         return (
-                            <Card>
+                            <Card key={Math.random()}>
```

Comment:
The goal of keys is to provide _stability_ between renders, so you definitely don't want to use a random value.

`index` should work fine here, lacking a database-generated key.

Reply:
We tried using index, but when you remove a question from the middle of the array, the index associated with each question changes so the components carry over their state instead of getting the new question associated with that index. Effectively, the last question is removed no matter which one you were trying to delete.

Reply:
Some of that may be due to not recreating your list of questions, but that aside...

The other option is to assign a random value _once_ to the question (on create), and use that as the key. What you want to avoid is a different key every time.